components/header.tsx

```diff
@@ -82,7 +82,7 @@ export default function Header(props: { fixed?: boolean }) {
   const navigation: NavigationItem[] = [
     { type: "dropdown", content: "Products", items: productsItems },
     { type: "dropdown", content: "Community", items: communityItems },
-    { type: "link", content: "Supporters", href: "/supporters" },
+    { type: "section", content: "Supporters", id: "supporters", page: "/supporters" },
```

Comment:
Why this change? 🤔

Reply:
> Why this change? 🤔

Because I want to use the section properties defined in 'switch' statements. Otherwise I have to write those properties in 'link' case.

Reply:
@Tanuj1718 a link must be used for links, you should find a way around this without disturbing existing elements